operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -404,6 +408,14 @@ type ExternalConnectivityConfig struct {
 	// are also available. The set of available functions is limited to hermetic
 	// functions because template application needs to be deterministic.
 	EndpointTemplate string `json:"endpointTemplate,omitempty"`
+	// PortTemplate is a Golang template string that allows customizing each
+	// broker advertised port, e.g. "32092 + {{ .Index }}".
+	// If not provided, the port will be the same as the internal one defined
+	// in the listener/endpoint's configuration such as KafkaAPI.Port.
+	PortTemplate string `json:"portTemplate,omitempty"`
+	// NoPortExposure is an indication that tells the controller not to expose the node port
+	// in a Kubernetes service. This is useful when the port is exposed by other means.
+	NoPortExposure bool `json:"noPortExposure,omitempty"`
```

Comment:
What "other means" would expose a port?

`ExcludeFromService` might be a more clear indicator of what this field controls.